tree/ntuple/v7/src/RField.cxx

```diff
@@ -233,22 +233,23 @@ std::string ROOT::Experimental::Detail::RFieldBase::GetQualifiedFieldName() cons
 }
 
 ROOT::Experimental::RResult<std::unique_ptr<ROOT::Experimental::Detail::RFieldBase>>
-ROOT::Experimental::Detail::RFieldBase::Create(const std::string &fieldName, const std::string &typeName)
+ROOT::Experimental::Detail::RFieldBase::Create(const std::string &fieldName, const std::string &typeName,
+                                               const std::string &typeAlias)
```

Comment:
`typeAlias` should probably default to the name before type name normalization (which involves `typedef`/`using` decl resolution).  For a future PR though.